vanhouse/src/components/UserList.js

```diff
@@ -1,103 +1,90 @@
-import {Card,Button,Container,Row,Col} from 'react-bootstrap';
-import React, {useState} from 'react';
-import user1 from '../assets/img1.jpg';
-import user2 from '../assets/img2.jpg';
-import user3 from '../assets/img3.jpg';
+import PropTypes from "prop-types";
+import React, { useEffect, useState } from "react";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import user1 from "../assets/img1.jpg";
+import user2 from "../assets/img2.jpg";
+import user3 from "../assets/img3.jpg";
 
+function UserList({ setFilterPost }) {

```

Comment:
@tyxxxx I found we can only see all posts by refreshing page after clicking on a username. We need to find a way to see all posts again.